### DISCUSSION TITLE
Best practices for teachers: demonstrating student interface?

### DISCUSSION THREAD

**Discussion by kbuffardi** [community/community#27395] [GitHub Education]
I have been using GitHub in my CS classes for years now, but I'm planning on integrating Codespaces (and autograding) into my class and making it their primary IDE. I've created a GitHub classroom, but want to be able to demonstrate students' interaction within the GitHub interface (e.g. accepting an invite to an assignment, using codespace to code/run/test, etc).

What is the best practice for teachers to show/preview how students will interact with the system? Is there a way to make a dummy student account or temporarily change my role from teacher to student?

**Comment by samus-aran** [comment]
Hey @kbuffardi 👋 
How have you been getting on with using Codespaces in your classroom(s)? As you're only able to have one type of account (faculty or student) there isn't currently a way to navigate between the different types of Classroom interfaces (teacher/student).  There are a number of videos demonstrating the Classroom interface including a [student accepting an assignment](https://www.youtube.com/watch?v=ObaFRGp_Eko&list=PLIRjfNq867bewk3ZGV6Z7a16YDNRCpK3u&index=5) and [documentation](https://docs.github.com/en/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/using-github-codespaces-with-github-classroom#about-the-codespaces-education-benefit-for-verified-teachers) on how to integrate Codespaces but not an end-to-end student journey including codespaces.
